### DISCUSSION TITLE
vue-tips/emit-event-from-composable

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#12] [Announcements]
# vue-tips/emit-event-from-composable

Your application may contain a lot of components that are using the same logic, like emitting the same event. In this case, it might be tempting to create a composable that will emit this event and is used in all components. But how do you emit an event from inside a composable?

https://mokkapps.de/vue-tips/emit-event-from-composable

**Comment by craiggeil** [comment]
Too bad you decided to do this in typescript :(

**Reply by Mokkapps** [reply]
Why? It shouldn't work the same way without TypeScript.
